backend/fms_core/migrations/0027_v3_5_0.py

```diff
@@ -0,0 +1,68 @@
+from django.db import migrations, models
+import django.db.models.deletion
+import re
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('fms_core', '0026_v3_5_0'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FullSample',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(help_text='Sample name.', max_length=200, validators=[django.core.validators.RegexValidator(re.compile('^[a-zA-Z0-9.\\-_]{1,200}$'))])),
+                ('alias', models.CharField(help_text='Sample alias.', max_length=200)),
+                ('volume', models.DecimalField(decimal_places=3, help_text='Current volume of the sample, in µL. ', max_digits=20)),
+                ('concentration', models.DecimalField(blank=True, decimal_places=3, help_text='Concentration in ng/µL. Required for nucleic acid samples.', max_digits=20, null=True, verbose_name='concentration in ng/µL')),
+                ('depleted', models.BooleanField(default=False, help_text='Whether this sample has been depleted.')),
+                ('collection_site', models.CharField(help_text='The facility designated for the collection of samples.', max_length=200)),
+                ('tissue_source', models.CharField(blank=True, help_text='Can only be specified if the biospecimen type is DNA or RNA.', max_length=200)),
+                ('experimental_group', models.JSONField(blank=True, help_text="Experimental group associated to  sample.")),
+                ('creation_date', models.DateField(help_text='Date of the sample reception or extraction.')),
+                ('coordinates', models.CharField(blank=True, help_text='Coordinates of the sample in a parent container. Only applicable for containers that directly store samples with coordinates, e.g. plates.', max_length=10)),
+                ('container', models.ForeignKey("Container", on_delete=models.DO_NOTHING, help_text="Container associated to the sample.")),
+                ('individual', models.ForeignKey("Individual", blank=True, null=True, on_delete=models.DO_NOTHING, help_text="Individual associated with to sample.")),
+                ('sample_kind', models.ForeignKey("SampleKind", on_delete=models.DO_NOTHING, help_text="Sample Kind associated to the sample.")),
+                ('derived_sample', models.ForeignKey("DerivedSample", on_delete=models.DO_NOTHING, help_text="Sample from which the current sample was extracted from.")),
+                ('biosample', models.ForeignKey("Biosample", on_delete=models.DO_NOTHING, help_text="Root Sample")),
+                ('child_of', models.ForeignKey("Sample", on_delete=models.DO_NOTHING, help_text="Parent Sample")),
+                ('process_measurements', models.ForeignKey("ProcessMeasurement", on_delete=models.DO_NOTHING, help_text="Process measurements associated to the sample")),
```

Comment:
This is unlikely to work ... each sample can have many process measurement related to it ...